internal/operands/common-templates/reconcile.go

```diff
@@ -90,6 +111,16 @@ func (c *commonTemplates) Reconcile(request *common.Request) ([]common.Reconcile
 	return append(reconcileTemplatesResults, oldTemplatesResults...), nil
 }
 
+func getClusterArchitectures(sspObj *ssp.SSP) []string {
+	if sspObj.Spec.EnableMultipleArchitectures != nil && *sspObj.Spec.EnableMultipleArchitectures {
```

Comment:
Same here? Check for length?

Reply:
`EnableMultipleArchitectures` is `*bool`, not a slice.

Reply:
done